### PR TITLE
Feedback for invalid Hex placement

### LINKED ISSUE #1397: Provide feedback on organelle place failure
A "blocked" sound should be played. Additionally when the problem is not enough MP the MP balance bar should be flashed.
When the problem is not touching an existing organelle then just play the sound.

### PR DESCRIPTION
**Pull request by CoreyHendrey**
Closes #1397

Waiting for an invalid placement sound.

### CODE REVIEW

**Comment by CoreyHendrey** [comment]
Hm, I feel like something has gone wrong with the MicrobeEditor.tscn, is this normal?

**Comment by hhyyrylainen** [comment]
It's pretty normal that the IDs get entirely remapped. It's not serious if nothing has broken.
Not sure if there is a way to avoid Godot inserting a resource in the middle of the ID sequencethat causes this

**Comment by hhyyrylainen** [comment]
Not sure if the shader is actually needed. I think there is a color_modulation property on all controls that can probably be animated.

**Comment by CoreyHendrey** [comment]
> Not sure if the shader is actually needed. I think there is a color_modulation property on all controls that can probably be animated.

Ah, right! That worked great. Removed the shader and I am now animating the modulate property.

**Comment by hhyyrylainen** [comment]
I saw that the missing sound was posted on discord https://cdn.discordapp.com/attachments/464847046608683029/741655016703524884/click_place_blocked.wav

File: src/microbe_stage/editor/MicrobeEditor.tscn [hunk 1]
```diff
@@ -86,6 +86,9 @@
...
...91, 213, 181, 241, 221, 29, 234, 201, 198, 234, 5, 17, 91, 183, 219, 98, 123, 109, 123, 55, 102, 116, 238, 158, 122, 39, 239, 235, 110, 205, 113, 125, 199, 223, 172, 134, 222, 218, 169, 75, 89, 52, 55, 177, 239, 37, 232, 155, 169, 1, 75, 241, 238, 212, 90, 140, 20, 158, 214, 150, 190, 88, 46, 87, 49, 118, 223, 231, 123, 125, 45, 44, 185, 64, 42, 16, 179, 113, 233, 118, 118, 44, 236, 5, 93, 98, 107, 44, 93, 86, 211, 165, 109, 145, 230, 191, 50, 119, 150, 150, 245, 220, 243, 86, 122, 250, 120, 182, 182, 162, 227, 221, 142, 62, 39, 23, 182, 110, 39, 166, 195, 122, 200, 254, 159, 199, 66, 94, 197, 182, 26, 31, 62, 55, 209, 34, 47, 38, 191, 242, 191, 216, 89, 55, 63, 245, 230, 203, 83, 164, 103, 251, 193, 92, 166, 85, 146, 55, 240, 196, 225, 30, 18, 238, 5, 10, 18, 67, 33, 105, 248, 97, 168, 108, 179, 52, 211, 193, 185, 144, 244, 123, 34, 0, 251, 117, 182, 148, 245, 252, 84, 249, 135, 5, 127, 170, 103, 235, 105, 148, 58, 215, 205, 79, 255, 86, 152, 43, 79, 54, 58, 243, 118, 207, 183, 94, 19, 72, 158, 204, 169, 144, 1, 222, 50, 94, 53, 252, 247, 168, 111, 16, 192, 6, 170, 134, 255, 30, 245, 13, 2, 216, 0, 41, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16, 129, 209, 70, 136, 253, 38, 196, 209, 20 )
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/MicrobeEditor.tscn:90]: the data should not be embedded in the scene file.

**Comment by CoreyHendrey** [comment]
> Should the bar flash when there isn't enough MP? In the segmented MP bar, the bar is red if the organelle that is about to be placed can't be afforded. You should choose which one to use when you resolve your merge conflict.

I'm thinking it'll use both. The red when it's about to be placed, but it will flash if the user actually clicks to attempt a placement.

**Comment by Kasterisk** [comment]
So I went and updated the PR, hopefully Nokk won't mind. I think this is ready to be merged

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**